### ISSUE TITLE
Add continuous integration

### ISSUE THREAD

**Issue by LOZORD** [UW-UPL/UPL-MMORPG#25]
**STRETCH GOAL**

Once testing #12 is done in the future, we should continuously integrate (using something like Travis) and deploy this game somewhere (on a UPL server?) so that people can play whenever (like a real MMO).

**Comment by jdetter** [comment]
That would be really cool. The basic networking stuff is already complete so it would be cool if we could get something up and running on nethack before the semester starts.
